editoast/src/tests/example_rolling_stock_1.json

```diff
@@ -1,5 +1,6 @@
 {
     "version": "3.1",
+    "locked": false,
```

Comment:
As we now have a distinct schema, RollingStockFull, for the front endpoints, and since this is used for endpoint testing, it makes sense to me to add locked to the examples (just like metadata is also in the examples). If you don't agree, i can always post add it in fixtures.rs.